src/modules/dashboard/components/ActionsPage/ActionsComponents/MintTokenMotion.tsx

```diff
@@ -62,6 +62,7 @@ const MintTokenMotion = ({
 }: Props) => {
   const motionTag = MOTION_TAG_MAP[MotionState.Motion];
   const passedTag = MOTION_TAG_MAP[MotionState.Passed];
+  const revealTag = MOTION_TAG_MAP[MotionState.Reveal];
```

Comment:
I think we might need to find a better place for these tags.

But not now, maybe we'll refactor at the end, after all is implemented.